app/models/invoice_subscription.rb

```diff
@@ -21,19 +32,19 @@ def fees
   end
 
   def from_datetime
-    fees_datetime('from_datetime')&.to_datetime
+    properties['from_datetime']&.to_datetime
```

Comment:
I'm not sure if I'm missing something, but I cannot see where does `properties` reference is coming from 🤔